pykern/pkio.py

```diff
@@ -125,6 +125,30 @@ def has_file_extension(filename, to_check):
     return e in to_check
 
 
+def is_binary(filename, threshold=0.30):
```

Comment:
I don't think threshold is important. The routine should be magic. If we need to parameterize, it's not magic enough. For example, we could read more data.

is_binary should be a [sans I/O](https://news.ycombinator.com/item?id=12242628) function in pykern.util and called by pkio for a convenient wrapper.

I think the test should be called `is_pure_text` or something like that. `is_binary` has many meanings, e.g. has executable bit set.